### DISCUSSION TITLE
Licensing for commercial use

### DISCUSSION THREAD

**Discussion by Helloman892** [SixLabors/ImageSharp.Web#283] [Q&A] [answered]
Hi there, I work as a developer in a commercial company with a turnover above US$1M. Myself and a coworker have been looking at ImageSharp.Web to serve images from our ASP.NET Core 6 API. 

We have looked into licensing and have found conflicting information on whether we can use this library in our software or not. On this repository's README, we found this line:
> - ImageSharp.Web is licensed under the [Apache License, Version 2.0](https://opensource.org/licenses/Apache-2.0)
> - An alternative Commercial Support License can be purchased for projects and applications requiring support. Please visit https://sixlabors.com/pricing for details.

To me (coming from the open source world) this means that we can use this library in our software so long as we distribute the correct attributions where applicable and we don't expect support. However, on the [SixLabors pricing page](https://sixlabors.com/pricing/), we see the following:
>  If you are consuming any Six Labors libraries as a Direct Package Dependency for usage in Closed Source software in the capacity of a For-profit company/individual with more than 1M USD annual gross revenue you must purchase a Six Labors Commercial License.

This implies that we can *not*, in fact, use ImageSharp.Web in our software without buying the $4999 Commercial License. We unfortunately do not have space in the budget for this license (in an ideal world, but management... you know how it is).

Which source should we take as true? We do not anticipate needing support with the library, so to my understanding (per this repository's README) we can use the library by following the Apache License 2.0 terms and nothing more.

Thanks,
Dylan G.

**Comment by JimBobSquarePants** [comment, marked as answer]
Hi Dylan,

Please see https://sixlabors.com/posts/license-changes/

We're changing the license for ImageSharp.Web V3+ (Not planning to release that until the end of the year). For V2 I was selling support licenses but no-one was buying them so we made them mandatory for the next major version under certain circumstance.

I've actually updated the price page. There's now two pricing tiers.

 So... For V2 you can use it just fine, once V3 comes out you must pay to use the upgraded version if your revenue is more than 1M USD.

**Reply by Helloman892** [reply]
Hi James,

Thanks a lot for your help, and for updating the pricing page with the new pricing tier!
My team will be glad to hear that we can use V2.
